fr/news/_posts/2014-12-25-ruby-2-2-0-released.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: news_post
+title: "Ruby 2.2.0 Released"
+author: "naruse"
+translator: "Geoffrey Roguelon"
+date: 2014-12-25 09:00:00 +0000
+lang: fr
+---
+
+Nous avons le plaisir de vous annoncer la sortie de Ruby 2.2.0.
+
+Ruby 2.2 contient beaucoup de nouvelles fonctionnalités et d'améliorations pour
+répondre aux diverses demandes qui sont toujours de plus en plus nombreuses.
+
+Par exemple, le ramasse miettes de Ruby peut désormais désallouer les objets Symbol.
+Ceci permet de diminuer l'usage méoire des symboles. Avant Ruby 2.2, les symboles
+n'étaient jamais désallouer.
+De plus Rails 5.0 nécessitera le ramasse mièttes des symboles et n
```

Comment:
@GRoguelon This and the next line seems to be some duplication/remainder from rc1 post?